packages/covector/package.json

```
@@ -14,7 +14,9 @@
     "covector": "./bin/covector.js"
   },
   "files": [
-    "bin/*"
+    "bin/*",
+    "dist/*.js",
+    "dist/*.d.ts"
```

Just including the directories should be enough here as they will be recursively included through more than one level of directory. Also, it feels as though anything in `dist` should be.... well... dist'd so even non JS and TS assets should be zipped up along side. A great example is sourcemaps which wouldn't be included in this list.

```suggestion
    "bin", 
    "dist"
```

---

I had that in the other (broken) PR but honestly how often do users call node scripts with sourcemaps enabled? Isn't shipping sourcemaps kinda wasted bandwidth? But it's not my project so I accept any choice!

---

I think the more appropriate action here is to remove the `files` property completely. The `bin` directory will be added by the `bin` property and the `dist` folder will be added by association with `main: "dist/index.js"`. Specifying the `files` property right now is clobbering those defaults.

---

@jbolda will this distribute sourcemaps? IMHO, sourcemaps are normally not needed, but when they are not there, it really stinks.

---

@cowboyd yea it will do everything in `dist`. I think this was originally the intention behind @dagda1's change.

```shell
npm notice
npm notice 📦  covector@0.6.0
npm notice === Tarball Contents ===
npm notice 377.3kB __snapshots__/index.test.ts.snap
npm notice 17.0kB  CHANGELOG.md
npm notice 11.1kB  README.md
npm notice 188B    bin/covector.js
npm notice 193B    dist/cli.d.ts
npm notice 250B    dist/cli.d.ts.map
npm notice 2.0kB   dist/cli.js
npm notice 127B    dist/index.d.ts
npm notice 225B    dist/index.d.ts.map
npm notice 401B    dist/index.js
npm notice 184B    dist/init.d.ts
npm notice 179B    dist/init.d.ts.map
npm notice 14.5kB  dist/init.js
npm notice 1.3kB   dist/run.d.ts
npm notice 1.4kB   dist/run.d.ts.map
npm notice 13.4kB  dist/run.js
npm notice 35.3kB  index.test.ts
npm notice 27B     modules.d.ts
npm notice 1.0kB   package.json
npm notice 1.8kB   src/cli.ts
npm notice 90B     src/index.ts
npm notice 12.7kB  src/init.ts
npm notice 13.3kB  src/run.ts
npm notice 155B    tsconfig.json
npm notice 245.7kB tsconfig.tsbuildinfo
npm notice === Tarball Details ===
npm notice name:          covector
npm notice version:       0.6.0
npm notice filename:      covector-0.6.0.tgz
npm notice package size:  80.3 kB
npm notice unpacked size: 749.7 kB
npm notice shasum:        24960a76cb50c87846bba609e8e4b26576bed7b0
npm notice integrity:     sha512-0hm8pea06o/6u[...]33WWZ/6yhvfIw==
npm notice total files:   25
npm notice
+ covector@0.6.0
```